README.md

```diff
@@ -248,44 +159,26 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project star the repo
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+We would like to thank the Microverse community for giving me the opportunity of learning with them.
+I would like like to extend more thanks to my code reviewer who have been there for me guiding in the day today activies to improve my technical skills in development
+More thanks goes to my coding partner, morning sessions and standup teams who have been getting me out of  blockages when i needed help and guidance. 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](MIT.md) licensed.
```

Comment:
- [ ] @mumbereh Great work trying to fix the MIT license link, however, its still not working, This is because the file you are referencing is not there, instead of ` [MIT](MIT.md)` it should be `[MIT](./src/MIT.md)`